_build/html/_sources/capstone-report.md

```diff
@@ -104,12 +107,9 @@ name: true_pos_all_wrong
 ---
 A manual inspection of tricky examples was conducted. Above, we have a true positive and although, all model are struggling, DenseNet is the least confident in its wrong prediction.  
 ```
+In an effort to make the model more generalizable, we considered various techniques such as implemeting dropout layers in our CNN structure and taking an ensemble approach. Further, the medical nature of this project suggests that reducing false negatives (optimizing recall) may be more important than maximizing the overall accuracy on postive and negative predictions. To find more about these explorations, see [Appendix B - Documentation](../documentation.md).
 
-In an effort to reduce the generalization error and high confidence in misclassifications, we considered adding dropout layers in our CNN structure. Ideally, dropout layers drop nodes in the network such that the model learns more robustly and the validation performance is improved. Another way to reduce this error is to average the predictions from all four models, called an ensemble. However, an ensemble would not be feasible as it requires lots of resources such as enough CPU to train the models and in our case, since the models performed too similarily, the average accuracy would be lower. Although the dropout layers did manage to reduce misclassifications in the models, again the overall reduced accuracy was not remarkable enough to implement those layers in our optimal model. To see this exploration, click [here](https://github.com/UBC-MDS/capstone-gdrl-lipo/blob/master/notebooks/densemodels-ax-dropout-layers.ipynb).
-
-Furthermore, as our capstone partner was far more interested in reducing false negatives, areas where insulin should not be injected, we investigated various techniques to optimize recall. Positive weight (pos_weight) is one argument of the loss function used in our CNN model which, if greater than 1, prioritizes the positive examples more such that there is a heavier penalization (loss) on labelling the positive lipohypertrophy examples incorrectly. However, this method was not implemented in our final model as it proved to be unstable. After conducting a few experiments, we found high variance in the [results](https://github.com/UBC-MDS/capstone-gdrl-lipo/blob/master/notebooks/pos-weight-exploration.ipynb). 
-
-Finally, the choice of our optimal model as DenseNet was further motivated by its size and compatibility with our data product, further discussed in the data product section.
+Finally, the choice of our optimal model as DenseNet was further motivated by its relaively small computational size as seen in the table below, and its compatibility with our data product, further discussed in the data product section.
```

Comment:
Replace "the table" with {numref}`size_df`

Replace "the data product section" with {ref}`data-product`